interactive_ai/tests/e2e/README.md

```diff
@@ -0,0 +1,48 @@
+**TODO: CVS-160717 move this information to the internal documentation**
+
+# e2e BDD tests
+
+## Setup
+
+### Install requirements
+
+1. Install nvm https://github.com/nvm-sh/nvm
+2. Make sure you use nvm 16 `nvm install 16` then `nvm use 16`
+3. Install swagger CLI: `npm install -g @apidevtools/swagger-cli`
+4. Install OpenAPI Generator CLI: `npm install @openapitools/openapi-generator-cli -g` (requires Java 11 or higher `sudo apt install default-jre` then `java -version`)
+
+### Obtain a Personal Access Token
+
+In the UI, go to the 'Account' section, select the 'Token' tab and click on 'Create' to generate an access token.
+
+### Setup the environment
+
+Set the following environment variables:
+
+- `GETI_SERVER_URL` -> address of the Geti server
+- `GETI_API_KEY` -> your personal access token
+
+The S3 credentials are required to download the test data; if you already have the data cached locally, you can skip setting these variables.
+If you don't know the credentials, ask the team.
```

Comment:
This part needs to be updated in scope of [ITEP-31960](https://jira.devtools.intel.com/browse/ITEP-31960), since the new test bucket is supposed to be public.